dbms/src/TiDB/Schema/tests/gtest_table_info.cpp

```
@@ -0,0 +1,435 @@
+String createTableStmt(
+    KeyspaceID keyspace_id,
+    DatabaseID database_id,
+    const TableInfo & table_info,
+    const SchemaNameMapper & name_mapper,
+    UInt64 tombstone,
+    const LoggerPtr & log);
+
+namespace tests
+{
+
+struct ParseCase
```

Clang (14.0.6)

[error] constructor does not initialize these fields: table_info_json, check

(cppcoreguidelines-pro-type-member-init,-warnings-as-errors)